src/diffpy/utils/diffraction_objects.py

```diff
@@ -537,6 +537,14 @@ def dump(self, filepath, xtype=None):
             The filepath where the diffraction object will be dumped
         xtype : str, optional, default is q
             The type of quantity for the independent variable chosen from {*XQUANTITIES, }
+
+        Examples
+        --------
+        To save a diffraction object to a file named "diffraction_data.chi" with
+        the independent variable 'q':
+
+        >>> file = "diffraction_data.chi"
```

Comment:
Let's also have another example, that shows a relative path.  I think that should then be clear enough.

Reply:
Sounds great! I'll add that.